pallets/structure/src/lib.rs

```diff
@@ -235,6 +330,7 @@ impl<T: Config> Pallet<T> {
 		})
 	}
 
+	/// Nests `token_id` under `owner` token
```

Comment:
I'm not sure what is the key difference between this method and ```nest_if_sent_to_token```? When should we use one or another?

Reply:
Added some comments to `nest_if_sent_to_token_unchecked` method